ftp-connectors/README.md

```diff
@@ -23,45 +20,61 @@ Your data format and transport can be set in your company integration settings b
 
 ### Protocol
 
-| Protocol | Description | Port |
-| :--- | :--- | :--- |
-| FTPS | Explicit FTP over TLS | 21 |
-| SFTP | SSH File Transfer Protocol  | 22 |
-| FTP w/o encryption | &lt;not supported&gt; |  |
+| Protocol | Description | Supported | Port |
+| :--- | :--- | :--- | :--- |
+| SFTP | SSH File Transfer Protocol  | Yes | 22 |
+| FTPS | Explicit FTP over TLS | No | |
+| FTP  | w/o encryption | No | |
 
 ### Username / password
 
-Username and password can be set in your company integration settings by your company admin
+The customer FTP username and password will be set by Tradecloud support.
+
+## FTP Folders
+
+There is one FTP folder structure per customer:
+```
+/order
+/order_change
+/order_response 
+/despatch_advice
+/receipt_advice
+/archive/order
+        /order_change
+        /order_response 
+        /despatch_advice
+        /receipt_advice
+```
 
 ## FTP Flows
 
 ### Uploading a file to Tradecloud
 
+{% hint style="warning" %}
+Both parties must use `0664` permission when uploading files.
+Setting the correct permissions is the responsibility of the client as outlined in the [standard](https://datatracker.ietf.org/doc/html/draft-ietf-secsh-filexfer-13#section-7.6).
+{% endhint %}
+
 ![](../.gitbook/assets/20180218-ftp-upload-flow-by-customer.png)
```

Comment:
Technically speaking, it's the polling that happens _after_ the file was uploaded that will cause step 3 and 4

Reply:
Improved